packages/zosuss/__tests__/__unit__/Shell.unit.test.ts

```diff
@@ -95,6 +112,13 @@ describe("Shell", () => {
         checkMockFunctionsWithCommand(command);
     });
 
+    it("Should execute ssh command when `$ exit` found", async () => {
```

Comment:
Do we need 2 separate unit tests or can they be combined into 1? I believe the only difference between them is that one calls `mockStream.emit("exit", 0);` and the other doesn't